packages/block-editor/src/components/block-tools/insertion-point.js

```diff
@@ -198,6 +198,50 @@ function InsertionPointPopover( {
 	const showInsertionPointInserter =
 		previousElement && nextElement && isInserterShown;
 
+	// Define animation variants for the line element.
+	const lineVariants = {
+		// Initial position starts from the center and invisible.
+		start: {
+			...( isVertical && { height: 0 } ),
```

Comment:
Nit: Felt a bit weird to see a check on each line. Could it be simpler to just have a separate object entirely for each case (`isVertical` false or true)